package-lock.json

```diff
@@ -1,8 +1,1869 @@
 {
   "name": "node-gcm",
   "version": "1.0.5",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
@p-kuen could you reinstate the original package-lock file and make any dependency install / upgrade without changing the lock file version from 1 to 2? See failing CI test.

Reply:
As of [the official documentation](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#lockfileversion), the lockfile version 2 is backwards compatible with lockfile version 1.

Additionaly, lockfile version 1 is generated by npm v6 which is eol according to [this answer](https://stackoverflow.com/a/66674123).

For this reason I would recommend to use npm v7 and lockfile version 2 as this PR requires already a major version change.

Reply:
We'd stay on v1 for other reasons regardless of claimed backwards compatibility, but you're right, this will require a major version increment, so we won't merge it for now. See https://github.com/ToothlessGear/node-gcm/pull/362#issuecomment-1200305406. Once we merge it, we can also set the lockfile version to 2.